config/environment.js

```diff
@@ -1,6 +1,14 @@
 /* eslint-env node */
 'use strict';
 
-module.exports = function(/* environment, appConfig */) {
-  return { };
+module.exports = function(environment/*, appConfig */) {
+  let ENV = {
+    APP: {}
+  };
+
+  if (environment === 'test') {
+    ENV.APP.autoboot = false;
+  }
```

Comment:
This change should go into `tests/dummy/config/environment.js` not this file

Reply:
Sorry about that. Moved.